scene/main/node.h

```diff
@@ -211,6 +217,10 @@ class Node : public Object {
 		bool display_folded = false;
 		bool editable_instance = false;
 
+		AutoTranslateMode auto_translate_mode = AUTO_TRANSLATE_MODE_INHERIT;
+		mutable bool is_auto_translating = true;
+		mutable bool is_auto_translate_dirty = true;
```

Comment:
This might be a silly question, but what makes these bools require `mutable` compared to other in the struct? Thread safety?

Reply:
See https://github.com/godotengine/godot/pull/87530#discussion_r1469625184.